### PR TITLE
Fix for eating cig_butts

### PR DESCRIPTION
**Pull request by karols146**
Closing #15931

### CODE REVIEW

**Comment by Coolthulhu** [comment]
It needs to lose the comestible-only fields too. For example, "comestible_type"

**Comment by Cyrano7** [comment]
Why you close this?

**Comment by karols146** [comment]
I just want to make other one with Coolthulhu ideas.

**Comment by Cyrano7** [comment]
Might be easier to just edit this one.

**Comment by karols146** [comment]
I know that just sound like a noob question, but how you can edit a opened merge ?

**Comment by Cyrano7** [comment]
Well if your using the graphical version of github there is a sync button in the top right. Just commit your changes to the branch you want to merge and push sync.

File: data/json/items/generic.json [hunk 1]
```diff
@@ -2473,7 +2464,7 @@
         "price": 2000,
         "material": "veggy",
         "weight": 420,
-        "volume": 4
+        "volume":"4"
```
Comment (Cyrano7) [inline data/json/items/generic.json]: This was fine just "volume": 4

**Comment by karols146** [comment]
I just try working on command line, and i just see that by "git commit -a" make new commit every time, don't get how to just edit commit with I just post. 
Work in Cygwin btw.

File: data/json/items/generic.json (hunk 1 above)
Comment (karols146) [inline data/json/items/generic.json]: Ok thanks for info, I don't even want to change that. Do it by mistake.

File: data/json/items/generic.json [hunk 2]
```diff
@@ -2473,7 +2464,7 @@
         "price": 2000,
         "material": "veggy",
         "weight": 420,
-        "volume": 4
+        "volume":4
```
Comment (Cyrano7) [inline data/json/items/generic.json]: You should change it back to the way it was actually. Your missing a space. Should look like "volume": 4

Comment (karols146) [inline data/json/items/generic.json]: Ok, right away,

File: data/json/items/generic.json [hunk 3]
```diff
@@ -2464,7 +2464,7 @@
         "price": 2000,
         "material": "veggy",
         "weight": 420,
-        "volume":4 
+        "volume": 4
```
Comment (karols146) [inline data/json/items/generic.json]: Like that ?

Comment (Cyrano7) [inline data/json/items/generic.json]: It still says there is a space behind the 4.

Comment (karols146) [inline data/json/items/generic.json]: > Should look like "volume": 4
You write it before, I must missing something here.

Comment (karols146) [inline data/json/items/generic.json]: Aaaa ok, Sorry for that, i just don't see "before" at first time.

File: data/json/items/generic.json [hunk 4]
```diff
@@ -2439,28 +2439,19 @@
...
-        "type" : "COMESTIBLE",
+        "type" : "GENERIC",
         "id" : "cig_butt",
         "name" : "cigarette butt",
         "weight" : 0,
         "color" : "dark_gray",
-        "addiction_type" : "nicotine",
-        "spoils_in" : 0,
-        "stim" : 1,
         "to_hit" : -2,
```
Comment (mugling) [inline data/json/items/generic.json]: Can also be dropped

File: data/json/items/generic.json [hunk 5]
```diff
@@ -2439,28 +2439,19 @@
...
-        "quench" : 0,
-        "healthy" : -3,
-        "addiction_potential" : 50,
-        "nutrition" : 0,
         "description" : "What was once a wonderfully addictive tube of dried tobacco leaf is now just a smelly piece of trash.  What a tragedy!\nThe leftover tobacco in a few of these could probably be used to roll another cigarette.  If you're willing to go that far...",
         "price" : 0,
         "tool" : "null",
         "volume" : 0,
         "charges" : 1,
         "stack_size" : 1000,
```
Comment (mugling) [inline data/json/items/generic.json]: Both `charges` and `stack_size` are redundant

File: data/json/items/generic.json [hunk 6]
```diff
@@ -2439,28 +2439,19 @@
...
         "to_hit" : -2,
-        "comestible_type" : "MED",
         "symbol" : "!",
-        "quench" : 0,
-        "healthy" : -3,
-        "addiction_potential" : 50,
-        "nutrition" : 0,
         "description" : "What was once a wonderfully addictive tube of dried tobacco leaf is now just a smelly piece of trash.  What a tragedy!\nThe leftover tobacco in a few of these could probably be used to roll another cigarette.  If you're willing to go that far...",
         "price" : 0,
         "tool" : "null",
```
Comment (mugling) [inline data/json/items/generic.json]: `tool` is only used by `COMESTIBLE`

Comment (karols146) [inline data/json/items/generic.json]: Ok i just think that tool is have something with, crafting, and that is a crafting ingredient, 
So thanks for info.

**Pull request merged**